src/main/java/com/gameshop/Controller/CategoryController.java

```diff
@@ -0,0 +1,47 @@
+package com.gameshop.Controller;
+
+import com.gameshop.entity.Category;
+import com.gameshop.exception.ResourceNotFoundException;
+import com.gameshop.repository.CategoryRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/category")
+public class CategoryController {
+    @Autowired
+    CategoryRepo categoryRepo;
+    @GetMapping("/category")
+    public List<Category> getAllCategory(){
+        return categoryRepo.findAll();
+    }
+
+    @PostMapping("/category/create")
```

Comment:
Just "/category"
Annotation @PostMapping already means that this is method to create